sklearn/naive_bayes.py

```diff
@@ -259,6 +278,9 @@ def _update_mean_variance(n_past, mu, var, X, sample_weight=None):
 
         return total_mu, total_var
 
+    @deprecated("The classes argument will be removed in version 0.21."
+                "A new classes argument has been added as a class parameter"
+                "since version 0.19 which is recommended to be used now.")
```

Comment:
This is not the right way to do the deprecation. You're deprecating the method. You should only deprecate the parameter. so if someone passes classes that is not None, inside the function you should raise a deprecation warning.